theme/overrides/home.html

```diff
@@ -63,12 +63,17 @@
 
       <!-- landing content -->
       <div class="tx-landing__content">
-        <h2>Make your development cycle <b>fast</b>, <b>efficient</b>, and <b>secure</b></h2>
-        <p>Cirrus CI is a modern Continuous Integration system built for the era of cloud computing.</p>
-        <p><a href="/guide/supported-computing-services">Bring your own infrastructure</a> or enjoy <a
-          href="/pricing/#compute-credits">per-second billing</a> with <b>no concurrency limits</b> on infrastructure
-          managed by us!</p>
-        <p><b>Build your Open Source projects on Linux, Windows, macOS and FreeBSD for free!</b></p>
+        <h2>Make your development cycle <b>fast</b>, <b>cost efficient</b>, and <b>secure</b></h2>
+        <p>Cirrus CI is arguably the most technological and flexible Continuous Integration service there is.</p>
+        <p>
+          Start on an infrastructure managed by us and enjoy <a href="/pricing/#compute-credits">per-second billing</a>
+          with <b>no concurrency limits</b>!
+        </p>
+        <p>
+          Grow your product without worrying about infrastructure maintenance and updates. When it's the right time,
+          configure and bring <a href="/guide/supported-computing-services">your own infrastructure</a>.
+        </p>
+        <p><b>Build your Open Source projects on Linux, Windows, macOS and FreeBSD for free including Arm options!</b></p>
```

Comment:
```suggestion
        <p><b>Build your Open Source projects on Linux, Windows, macOS and FreeBSD for free, including ARM architecture!</b></p>
```

Reply:
Good catch! Apparently Arm the company and ARM the architecture.